doc/classes/AStar.xml

```diff
@@ -157,6 +157,12 @@
 				If you change the 2nd point's weight to 3, then the result will be [code][1, 4, 3][/code] instead, because now even though the distance is longer, it's "easier" to get through point 4 than through point 2.
 			</description>
 		</method>
+		<method name="get_point_capacity" qualifiers="const">
```

Comment:
Could you add documentation for those new methods too?

Reply:
Will do!